criticality_score/run.py

```diff
@@ -608,6 +608,62 @@ def initialize_logging_handlers():
     logging.getLogger('').addHandler(console)
 
 
+def override_params(override_params):
```

Comment:
Would it make more sense to restructure `constants`/`defaults` to make it easier to replace the default values, rather than mutating the initial values?

e.g. turn the constants into a dict, where each value is a namedtuple or dataclass and pass the dict (with any overrides) as an argument into the scoring method.

Reply:
There always are many ways one can implement things and, especially when I'm new to a project, I try to keep my PRs to a minimum to make them easier to be accepted. I thought it'd be argued that rather than renaming constants.py defaults.py I should split the file in two and only put in defaults.py things that can indeed be modified. :-)

Reply:
In this case I don't mind the extra effort to improve the overall implementation. But I'm happy to approve this change as is as it doesn't change the things in a way that can't be changed to a `dict` based approach in a future PR.